app/src/main/java/com/infomaniak/swisstransfer/ui/MainApplication.kt

```diff
@@ -59,7 +59,7 @@ class MainApplication : Application(), Configuration.Provider {
 
         globalCoroutineScope.launch {
             accountUtils.init()
-            transferManager.deleteExpiredTransfers()
+            if (accountUtils.isUserConnected()) transferManager.deleteExpiredTransfers()
```

Comment:
Why this change ?

Reply:
`deleteExpiredTransfers` crashes when you start the app because it can't find any realm to read from when there's no user connected. And at this point in time, during your very first start of the app, you're not connected to any account yet